scopes/ui-foundation/ui/start.cmd.tsx

```diff
@@ -33,7 +33,7 @@ export class StartCmd implements Command {
   options = [
     ['d', 'dev', 'start UI server in dev mode.'],
     ['p', 'port [port-number]', 'port of the UI server.'],
-    ['r', 'rebuild', 'rebuild the UI'],
+    ['r', 'rebuild', 'rebuild the UI (useful e.g. when the bundler config has been changed in the env)'],
```

Comment:
you don't have to rebuild the ui if you changed env-webpack-config.
its useful when you are actually working and implementing features that are a part of the worksapce ui.